app/bundles/LeadBundle/Views/Lead/lead.html.php

```diff
@@ -458,9 +458,8 @@ class="arrow text-muted text-center<?php echo ('expanded' == $avatarPanelState)
                 $style = !empty($color) ? ' style="font-color: '.$color.' !important;"' : '';
                 ?>
                 <h1 <?php echo $style; ?>>
-                    <?php echo $view['translator']->transChoice(
+                    <?php echo $view['translator']->trans(
                         'mautic.lead.points.count',
-                        $lead->getPoints(),
                         ['%points%' => $lead->getPoints()]
```

Comment:
```suggestion
                        ['%count%' => $lead->getPoints()]
```
plus update the ini file